src/components/Tabber/Tabs/Tabs.tsx

```diff
@@ -1,30 +1,32 @@
-﻿import classNames from 'classnames';
+﻿import { ReactNode } from 'react';
+
+import classNames from 'classnames';
 
 import scss from '~components/Tabber/Tabs/Tabs.module.scss';
 
 import { TSocialNetworks } from '../stores/useSocialNetworkStore.types';
 import { ITabsProps } from './Tabs.types';
 
-function Tabs(props: ITabsProps) {
-  const tabClasses = (id: string) =>
+function Tabs(props: ITabsProps): ReactNode {
+  const tabClasses = (id: string): string =>
     classNames(scss.tab, id === props.currentTab.id && scss.active);
 
-  const renderTabs = (socialNetwork: TSocialNetworks) => (
-    <div
+  const renderTabs = (socialNetwork: TSocialNetworks): ReactNode => (
+    <button
       className={tabClasses(socialNetwork.id)}
       key={`${socialNetwork.id}-${socialNetwork.name}`}
       onClick={() => props.onChangeTab(socialNetwork)}
     >
       {socialNetwork.icon}
       <span className={scss.tabTitle}>{socialNetwork.name}</span>
-    </div>
+    </button>
   );
 
-  return (
-    <div className={scss.tabsContainer}>
-      {props.socialNetworks.map(renderTabs)}
-    </div>
+  const listSocialNetworks: ReactNode = props.socialNetworks.map(
+    (socialNetwork: TSocialNetworks) => renderTabs(socialNetwork)
   );
+
+  return <div className={scss.tabsContainer}>{listSocialNetworks}</div>;
```

Comment:
```suggestion
  return <div className={scss.tabsContainer}>{props.socialNetwroks.map(renderTabs)}</div>;
```

Sei que geralmente recomendo evitar `map` dentro de tags, mas esse é um caso conciso e mais importante não é multiline, evitamos a declaração de uma outra variável e mantemos tudo bem simples.